pkg/get/tools.go

```diff
@@ -957,5 +957,40 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}-{{.VersionNumber}}-{{.OS}}-{{$file}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "istio",
+			Repo:    "istio",
+			Name:    "istioctl",
```

Comment:
Shall we put `istioctl` on the homepage as a downloadable tool, too?

Reply:
Done